docs/docs_skyrim_platform.md

```diff
@@ -353,6 +355,39 @@ browser.executeJavaScript("console.log('Hello CEF')");
 const str = browser.getToken();
 ```
 
+#### Talking back to the game
+
+Use `window.skyrimPlatform.sendMessage` on the browser side to talk back to the game. `sendMessage` accepts zero or more JSON-serializable values.
```

Comment:
IDK if it's good or bad practice in frontend development, but feels like explicitly writing `window.` is not needed

Reply:
it matters, window object and global object are different